cloudinit/config/cc_disk_setup.py

```diff
@@ -827,6 +829,36 @@ def mkpart(device, definition):
     LOG.debug("Partition table created for %s", device)
 
 
+def block_until_file_exists(path, timeout=30):
```

Comment:
is this necessarily different from utils.wait_for_files ?

Reply:
I just wasn't aware of that function, I'll use it instead